lib/tasks/travis.rake

```diff
@@ -0,0 +1,9 @@
+require 'system_commands'
+
+namespace :travis do
+  desc "Update vendor/bundle files to gem cache on tmc-server"
+  task :gemcache do
+    SystemCommands.sh!("find vendor/bundle -iname '*.gem'  -print0  | rsync -0 --checksum -avz  --files-from=- . web01:/srv/www/testmycode.net/travis/rubygems-cache/", {escape: false})
```

Comment:
@mpartel not sure if you had thought something more general for this... This will at least simplify the rubygems part (though it is currently missing the install gems to vendor/bundle folder)